packages/rancher-monitoring/generated-changes/overlay/files/rancher/rancher-default-home.json

```diff
@@ -0,0 +1,1275 @@
+{
```

Comment:
Since I was moving dashboards out into their own files/ directory, I copied and pasted this out of the original ConfigMap it was in and moved it into a file that would reference it.

This seems to render fine on the Grafana UI.

![Screen Shot 2021-03-04 at 6 45 43 PM](https://user-images.githubusercontent.com/13127443/110059819-d4a26900-7d19-11eb-9c55-9d3c33b92cd5.png)